design.css

```diff
@@ -835,3 +836,24 @@ thead {
   font-size: 0.8rem;
   font-weight: normal; /* reset style global style */
 }
+
+.ocean-icon:after {
+  content: url("/ocean-logo.svg");
+  margin-left: 0.5rem;
+}
+
+.bdp-icon {
+  min-width: 20px;
+  max-width: 20px;
+  height: 20px;
+  background: url("/bdp-logo.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin-left: 0.5rem;
+}
+
+.content {
```

Comment:
Can we please rename this to something descriptive as e.g. "protocol-logo"?

Reply:
The content inside `<td>` is wrapped inside a div of this class. This `<td>` is the second column of the table and its heading is Content. That's why I have used it.

```html
<div class="content">
    <div>
      <a
        style="color: #ff4092"
        target="_blank"
        href="https://market.oceanprotocol.com/asset/did:op:7Bce67697eD2858d0683c631DdE7Af823b7eea38"
      >
        Annotated Images
      </a>
    </div>
    <div class="ocean-icon" title="Ocean Protocol"></div>
</div>
```

If the naming still doesn't feel right I'd be happy to change it.

Reply:
ah ok sorry I my bad. Then it's a separate issue.